### PR TITLE
NAS-115176 / 22.12 / snmp-agent integration test

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by themylogin** [comment]
time 02:00

**Comment by yocalebo** [comment]
Why change around imports?

**Comment by themylogin** [comment]
> Why change around imports?

https://peps.python.org/pep-0008/#imports

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
```

File: tests/api2/test_snmp_agent.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "1.3.6.1.4.1.50536",
+        shell=True,
+        check=True,
+        capture_output=True,
+        text=True,
+    )
+
+    assert "FREENAS-MIB::zpoolDescr.1 = STRING: boot-pool\n" in snmp.stdout
+    assert "FREENAS-MIB::datasetDescr.1 = STRING: boot-pool/.system\n" in snmp.stdout
+    assert re.search("^FREENAS-MIB::zfsArcSize\.0 = Gauge32: ([1-9][0-9]+)\n", snmp, re.MULTILINE), snmp.stdout
```
Comment (sonicaj) [inline tests/api2/test_snmp_agent.py:28]: Do we want to use a raw string here ?

**Comment by yocalebo** [comment]
> > Why change around imports?
> 
> https://peps.python.org/pep-0008/#imports
> 
> ```
> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> ```

If we're going to adhere to random PEPs, then we should enforce it at the CI level.

**Review by yocalebo — APPROVED** [review]

**Pull request merged**